Multisig/Data/Services/Safe Client Gateway Service/ProposeTransactionRequest.swift

```diff
@@ -9,8 +9,8 @@
 import Foundation
 
 struct ProposeTransactionRequest: JSONRequest {
-    let safe: AddressString
-    let sender: AddressString
```

Comment:
these things here should have been AddressString, I think. What I meant is only change the types of the function arguments. Is it works like this?